Gemfile

```diff
@@ -41,8 +39,7 @@ gem 'omniauth-rails_csrf_protection'
 
 gem 'simple_form'
 
-gem 'paperclip', '~> 5.2'
-gem 'aws-sdk', '~> 2.3.0'
+gem 'kt-paperclip', '~> 7.1'
```

Comment:
Original paperclip gem is deprecated and isn't compatible with Ruby 3.

Reply:
This should be fine for now. We should make a plan to migrate to active storage.

Reply:
There is one attachment type in this app, and we'll want to make sure that the newer version of paperclip can still find the previously-uploaded files in production.